lettuce/boundary.py

```diff
@@ -37,3 +38,63 @@ def __call__(self, f):
         feq = self.lattice.einsum("q,q->q", [feq, torch.ones_like(f)])
         f = torch.where(self.mask, feq, f)
         return f
+
+class AntiBounceBackOutlet:
+    """Allows distributions to leave domain unobstructed through this boundary.
+        Based on equations from page 195 of "The lattice Boltzmann method" (2016 by Krüger et al.)
+        Give the planned side of the boundary as array [1, 0, 0] for positive x-direction in 3D; [1, 0] for the sam in 2D
+        [-1, 0, 0] is negative x-direction ect. [x, y, z]; only one entry nonzero
+        """
+
+    def __init__(self, lattice, direction):
+        direction = np.array(direction)
```

Comment:
Assert if direction has the correct shape

Reply:
cue totally over complicated assert checking for all the possible errors (the function only accepts one kind of input, so i feel this is justified)

Reply:
It's well documented, that is enough for me.

Reply:
The assert is now pushed, so you can see if that is helpful or too much. I was hoping all the outputs would make it easy to fix errors?